### ISSUE TITLE
New Feature Bubble Elevator

### ISSUE THREAD

**Issue by OrionSincoat** [MongoTheElder/PITG#6]
is it possible to have 1 block bubble elevators?

**Comment by MongoTheElder** [comment]
Not really. The water column formed when source blocks are placed above soul sand or a magma block doesn't carry over to waterlogged blocks. The closest you can get is glass panes surrounding water source blocks but that is no longer a 1x1 structure.
I'll have to take a look at the vanilla mechanics to see if it is even possible for waterlogged blocks to make a water column.

**Comment by OrionSincoat** [comment]
i see, 
so it's impossible to tell that your waterlogged block is a source block.

Thanks!
And good work, i was in need of your mod feature since ever.

**Comment by MongoTheElder** [comment]
I'll leave this open until I have a chance to investigate making this work like a bubble column.
